layouts/New/layouts.module.css

```diff
@@ -141,27 +141,17 @@
 .blogLayout {
   @apply flex
     w-full
+    max-w-8xl
```

Comment:
Why did you add this? This broken the auto margins 🤔

Reply:
it's from [this line](https://github.com/nodejs/nodejs.org/pull/6350/files/88e9845728b2ccd013c9eff43fdc90534c48436a#diff-c4b4ff7c439381cc3dfccce4a24420a9b67403bbabb4e5302066fd07f6e272c1L151) is when I merged main and `blogLayout`.

Reply:
But... why did you merged them? I understand from which line it is, but could you please undo this change? (the merge?)

There's a reason why there were two different classes.

Reply:
I had merged 'cause on JSX we had something like that 
```tsx
<div className={styles.blogLayout}>
  <main>
    {...main}
  </main>
</div>
```